webapp-django/crashstats/signature/static/signature/js/signature_panel.js

```diff
@@ -0,0 +1,33 @@
+SignatureReport.Panel = function (panelName, onDelete) {
+
+    // For accessing this from inside functions.
+    var that = this;
+
+    // Make the HTML elements.
+    this.$panelElement = $('<div>', {'class': 'panel'});
+    var $headerElement = $('<div>', {'class': 'title'});
+    var $headingElement = $('<h2>', {
+        'text': SignatureReport.capitaliseHeading(panelName)
+    });
+    var $deleteButton = $('<a>', {
+        'href': '#',
+        'class': 'options delete',
+        'text': 'X'
+    });
+    var $bodyElement = $('<div>', {'class': 'body'});
+    this.$contentElement = $('<div>', {'class': 'content'});
+
+    // Bind the delete function to the delet button.
+    $deleteButton.on('click', function (e) {
+        e.preventDefault();
+        that.$panelElement.remove();
+        onDelete();
```

Comment:
It would be nice here to check if ``onDelete`` is a function, and to not call it if it's not. This way we can easily add new close-able panels in the future without requiring a callback.